client/components/ProfileHeader.js

```diff
@@ -0,0 +1,120 @@
+// Module imports
+import React from 'react'
+// import PropTypes from 'prop-types'
+
+
+
+
+// Component imports
+import { connect } from '../store'
+import { formatAsEliteDateLong } from '../helpers/formatTime'
+import {
+  selectUser,
+  selectUserDisplayRat,
+  selectUserAvatar,
+} from '../store/selectors'
+
+
+
+
+// Component Constants
+
+
+
+
+
+@connect
+class ClassName extends React.Component {
+  /***************************************************************************\
+    Class Properties
+  \***************************************************************************/
+
+  state = {}
+
+
+
+
+
+  /***************************************************************************\
+    Private Methods
+  \***************************************************************************/
+  _renderUserGroups = () => this.props.user.relationships.groups.data.map((item) => <li className={`badge ${item.id}`} key={item.id}>{item.id}</li>)
+
+
+
+
+  /***************************************************************************\
+    Public Methods
+  \***************************************************************************/
+
+  render () {
+    const {
+      displayRat,
+      userAvatar,
+    } = this.props
+
+    let { attributes } = this.props.user
```

Comment:
These 2 lines could actually be
```suggestion
    const attributes = this.props.user.attributes || {}
```